spec/models/mediaflux/store_list_request_spec.rb

```diff
@@ -2,10 +2,13 @@
 require "rails_helper"
 
 RSpec.describe Mediaflux::StoreListRequest, connect_to_mediaflux: true, type: :model do
-  let(:mediaflux_url) { "http://mflux-ci.lib.princeton.edu/__mflux_svc__" }
+  let(:mediaflux_url) { Mediaflux::Request.uri.to_s }
   let(:user) { FactoryBot.create(:user) }
   let(:approved_project) { FactoryBot.create(:approved_project) }
   let(:mediaflux_response) { "<?xml version=\"1.0\" encoding=\"UTF-8\" ?>\n<response><reply type=\"result\"><result></result></reply></response>" }
+  # Docker and Ansible responses are the same, but may be different in the future
+  let(:docker_response_store) { { id: "1", name: "db", tag: "", type: "database" } }
```

Comment:
Is it me or do these lines match.  Why the switch here?

Reply:
They are the same now, but I was leaving room for the potential that they may differ in the future. Maybe one build has something stored in the tag field? We could just go back to having one field for the store if that makes more sense